test/case/ietf_interfaces/bridge_fwd_dual_dut/test.py

```diff
@@ -1,38 +1,45 @@
 #!/usr/bin/env python3
-#    ,-------------------------------------,       ,-------------------------------------,
-#    |                          dut1:data2 |       | dut2:data2                          |
-#    |                      br0  ----------|-------|---------  br0                       |
-#    |                     /   \           |       |          /   \                      |
-#    |dut1:mgmt  dut1:data0     dut1:data1 |       | dut2:data0    dut2:data1  dut2:mgmt |
-#    '-------------------------------------'       '-------------------------------------'
-#        |                |     |                            |     |                 |
-#        |                |     |                            |     |                 |
-# ,-----------------------------------------------------------------------------------------,
-# |  host:mgmt0  host:data10    host:data11         host:data20    host:data21   host:mgmt1 |
-# |                             [10.0.0.2]          [10.0.0.3]     [10.0.0.4]               |
-# |                               (ns11)              (ns20)         (ns21)                 |
-# |                                                                                         |
-# |                                        [ HOST ]                                         |
-# '-----------------------------------------------------------------------------------------'
-"""
+r"""
 Bridge forwarding dual DUTs
 
 Ping through two bridges on two different DUTs.
+
+....
+
+   ,-------------------------------------,       ,-------------------------------------,
+   |                          dut1:data2 |       | dut2:data2                          |
+   |                      br0  ----------|-------|---------  br0                       |
+   |                     /   \           |       |          /   \                      |
+   |dut1:mgmt  dut1:data0     dut1:data1 |       | dut2:data0    dut2:data1  dut2:mgmt |
+   '-------------------------------------'       '-------------------------------------'
```

Comment:
Step up theese as well, so they map to the code

Reply:
fixed